ding/data/buffer/middleware/priority.py

```diff
@@ -1,24 +1,24 @@
 from typing import Callable, Any, List, Dict, Optional, Union
 import copy
 import numpy as np
+import torch
 from ding.utils import SumSegmentTree, MinSegmentTree
-from ding.worker.buffer.buffer import BufferedData
+from ding.data.buffer.buffer import BufferedData
 
 
 class PriorityExperienceReplay:
 
     def __init__(
             self,
             buffer: 'Buffer',  # noqa
```

Comment:
Use TYPE_CHECKING to load module, don't use noqa. Example: https://github.com/opendilab/DI-engine/blob/main/ding/utils/log_writer_helper.py#L8

Reply:
fixed it